doc/articles/getting-started/counterapp/includes/include-image-csharp.md

```diff
@@ -5,5 +5,5 @@
     new Image()
         .Width(150)
         .Height(150)
-        .Source("ms-appx:///Counter/Assets/logo.png")
+        .Source("ms-appx:///Assets/logo.png")
```

Comment:
Shouldn't we be keeping `Counter`?

Reply:
The structure change there is only one main project. But by saying that, this is not true anymore : 

"The template will create a new solution with a number of projects. The main project is called **Counter** which contains the application code. The other projects are platform-specific heads that contain the platform-specific code required to run the application on each platform."

I'll change that.

Reply:
Oh right I forgot about single project.

So with the single project, we no longer need to specify the project name in the ms-appx? Nice :)

Reply:
That's correct 😃